lib/rust/flexer/src/automata/alphabet_segmentation.rs

```diff
@@ -0,0 +1,82 @@
+//! Exports an alphabet for an arbitrary finite state automaton.
+
+use crate::automata::symbol::Symbol;
+
+use std::collections::BTreeSet;
+use std::ops::RangeInclusive;
+use enso_prelude::default;
```

Comment:
Few things:

1. `enso_prelude` is one of these rare crates that is designed to be imported as `::*`. We use it everywhere like that.
2. We have a common pattern where we write `pub use enso_prelude as prelude` in `lib.rs` and instead you can write everywhere `use crate::prelude::*` . This way you prepare also for a situation when in a crate you'd like to expose more things than `enso_prelude` - you could just extend that in `lib.rs` and you'd not need to change any imports in modules.
3. We sort imports - just use intellij plugin for sorting lines :)